src/page-home/HomePage.tsx

```diff
@@ -1,5 +1,5 @@
+import { Link } from "gatsby";
```

Comment:
Similar to my previous comment: Why did we switch all of these navigation components to something provided by Gatsby? At first/immediate glance, switching core functionality from React out to a more specific/implementation-driven set of components makes me _slightly_ nervous.

Reply:
React Router isn't an official React library, it's a third-party library just like Reach Router is.

It's probably a bad to use two routing libraries simultaneously, as I imagine they might manage their own state.